workflows/prognostic_c48_run/runtime/nudging.py

```diff
@@ -128,6 +131,8 @@ def _get_reference_state(
     if MPI.COMM_WORLD.rank == 0:
         fs = fsspec.get_fs_token_paths(dirname)[0]
         fs.get(dirname, localdir, recursive=True)
+        if restart_categories is not None:
+            _rename_local_restarts(localdir, restart_categories)
```

Comment:
I see how the renaming in needed since `fv3gfs.util.open_restart` hardcodes the name. At some point it would be nice to parameterize this hardcode upstream so that we can avoid doing additional I/O for something like this.

Reply:
Can you open an issue for that?

Reply:
Actually maybe it would be more performant to use symbolic/hard links and avoid the copy.